pkg/v1/providers/yttcb/clusterbootstrap.yaml

```diff
@@ -0,0 +1,129 @@
+#@ load("@ytt:data", "data")
+#@ load("lib/helpers.star", "get_tkr_version_from_tkr_name")
+
+#@ def is_antrea_configs_given():
+#@   return (data.values.NSXT_POD_ROUTING_ENABLED != False or
```

Comment:
Why are we doing string compare for boolean?

Reply:
Fixed